spec/helpers/chat_mirgation_message_spec.rb

```diff
@@ -0,0 +1,25 @@
+RSpec.describe ChatMigrationMessage, type: :helper do
+  describe '#display_message?' do
+
+    context 'before display_message date' do
+      it 'returns false' do
+      	allow(Date).to receive(:today) { Date.new(2016, 2, 2) }
+        expect(display_chat_migration_message?).to be false
+      end
+    end
+
+    context 'on display_message date' do
+      it 'returns true' do
+        allow(Date).to receive(:today) { Date.new(2016, 2, 6) }
+        expect(display_chat_migration_message?).to be true
+       end
+     end
+
+     context 'after display_message date' do
+       it 'returns false' do
+         allow(Date).to receive(:today) { Date.new(2016, 2, 10) }
+         expect(display_chat_migration_message?).to be false
+       end
+     end
+  end
+end
```

Comment:
Really like testing before the 6th, on the 6th and after.

The condition (only on the 6th) would be a bit clearer and tighter by making the :before date the 5th, and the :after date the 7th. The reason being that looking only at the test my first though is what happens on the dates between the 2nd and 6th, and between the 6th and the 10th?